src/index.js

```diff
@@ -100,24 +134,60 @@ module.exports = {
       type: 'covariates',
     }],
   }, {
+    // step two: receive the globalMeanY and averageBetaVector, then calculate sseLocal, sstLocal and varXLocal
     type: 'function',
     fn(opts) {
-      const previousData = opts.previousData;
+      const globalMeanY = opts.remoteResult.data.globalMeanY;
+      const averageBetaVector = opts.remoteResult.data.averageBetaVector;
+      const biasedX = opts.previousData.biasedX;
+      const y = opts.previousData.y;
+      const rSquared = opts.previousData.rSquared;
+      const tValue = opts.previousData.tValue;
+      const pValue = opts.previousData.pValue;
+      const betaVector = opts.previousData.betaVector;
+      const localCount = y.length;
 
-      for (let i = 0; i < opts.previousData.X.length; i += 1) {
-        previousData.X[i].splice(0, 0, 1);
-      }
+      //calculate the local r squred and t value for averageBetaVector)
```

Comment:
This comment and ones following don’t add meaning to the code: the variable and function names should explain this. Please drop.